apps/studio/components/interfaces/Settings/Logs/Logs.utils.ts

```diff
@@ -149,6 +162,16 @@ export const genDefaultQuery = (table: LogsTableName, filters: Filters) => {
   `
 
     case 'postgres_logs':
+      if (IS_PLATFORM === false) {
+        return `
+select id as identifier, postgres_logs.timestamp, id, event_message, parsed.error_severity
```

Comment:
```suggestion
select postgres_logs.timestamp, id, event_message, parsed.error_severity
```

id is not identifier, we can remove the `identifier field completely.